src/coreclr/vm/dbginterface.h

```diff
@@ -72,11 +72,6 @@ class DebugInterface
                             BOOL         fAttaching) = 0; // true if this notification is due to a debugger
                                                           // being attached to the process
 
-    // Called AFTER LoadModule, and after the module has reached FILE_LOADED. This lets
-    // dbgapi do any processing that needs to wait until the FILE_LOADED stage (e.g.,
-    // binding breakpoints in NGENd generics).
-    virtual void LoadModuleFinished(Module * pModule, AppDomain * pAppDomain) = 0;
```

Comment:
This is a breaking interface change. Is that okay or do we need a placeholder here?

Reply:
Based on the comments at the top of this header, this API is only used internally within coreclr.dll/libcoreclr.so. Now that the debugger implementation is in the same binary as CoreCLR, this interface doesn't need to deal with versioning any more.